react/features/chat/components/native/Chat.js

```diff
@@ -58,7 +61,9 @@ class Chat extends AbstractChat<Props> {
                 style = { styles.chatContainer }>
                 <MessageContainer messages = { _messages } />
                 <MessageRecipient privateMessageRecipient = { privateMessageRecipient } />
-                <ChatInputBar onSend = { this._onSendMessage } />
+                <KeyboardAvoidingView behavior = { Platform.OS === 'ios' ? 'padding' : 'height' }>
```

Comment:
This is the only solution that I have found. And it seems that the problem still persists on react navigation side
https://github.com/react-navigation/react-navigation/issues/10151

Reply:
This is an acceptable solution!

Reply:
I think i've found the problem. I will update the PR soon.